library/Notifications/Widget/Calendar/WeekGrid.php

```diff
@@ -118,8 +118,8 @@ protected function createSidebar(): BaseHtmlElement
 
     protected function assembleGridStep(BaseHtmlElement $content, DateTime $step): void
     {
-        if ($step->format('H') === '23') {
-            $dayViewUrl = $this->calendar->prepareDayViewUrl($step);
+        $dayViewUrl = $this->calendar->prepareDayViewUrl($step);
```

Comment:
If you'd define this inside the first `if`, the url wouldn't be cloned and whatnot in 99% (23 times per day) of the time.